src/components/OSCALControlProse.js

```diff
@@ -267,22 +278,34 @@ function getParameterValueSegment(
     parameterId
   );
 
-  if (!constraintsDisplay.length) {
+  const parameterLabelText = getParameterLabelText(
+    parameters,
+    parameterId,
+    modificationSetParameters
+  );
+
+  if (parameterValue) {
+    if (!constraintsDisplay.length) {
+      return styledParamValue(`param-value-key-${key}`, parameterValue);
+    }
     return (
-      <ParamValue component="span" key={`param-value-key-${key}`}>
-        {parameterValue}
-      </ParamValue>
+      <SegmentTooltipWrapper
+        constraintsDisplay={constraintsDisplay}
+        key={`segment-wrapper-key-${key}`}
+      >
+        {styledParamValue(`param-value-key-${key}`, parameterValue)}
+      </SegmentTooltipWrapper>
     );
   }
-
+  if (!constraintsDisplay.length) {
+    return styledParamLabel(`param-label-key-${key}`, parameterLabelText);
+  }
   return (
     <SegmentTooltipWrapper
-      constraintsDisplay={constraintsDisplay}
+      constraintDisplay={constraintsDisplay}
```

Comment:
Maybe I'm missing something but why is the prop here named `constraintDisplay` and not `constraintsDisplay` as in other places?

Reply:
I must have mistyped! It should be `constraintsDisplay` and not `constraintDisplay`. My most recent commit now reflects this change.